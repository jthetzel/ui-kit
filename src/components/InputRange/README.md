InputRange

```js
initialState = { value: 0 };

const onChange = (event) => {
  setState({ value: event.target.value })
};

<div>
  Current value: {state.value}
  <InputRange
    onChange={onChange}
    value={state.value}
  />
</div>
```