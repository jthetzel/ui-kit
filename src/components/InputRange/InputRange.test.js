import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import InputRange from '../InputRange';

const onChange = value => value;

test('Slider does not crash', () => {
  const wrapper = shallow(<InputRange onChange={onChange} />);
  expect(toJson(wrapper)).toMatchSnapshot();
});
