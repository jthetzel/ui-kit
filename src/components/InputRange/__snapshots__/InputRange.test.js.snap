// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Slider does not crash 1`] = `
<Styled(input)
  onChange={[Function]}
  type="range"
/>
`;
